python/GafferUI/CompoundEditor.py

```diff
@@ -730,11 +730,34 @@ def __removePanel( self ) :
 #  - Manage hover states during a drag operation.
 #  - Coordinate the re-parenting of Tab widgets.
 #  - Manage the life-cycle of detached panels affected by the drag.
+#  - Correct the cursor appearance as much as possible.
 #
 class _TabDragBehaviour( QtCore.QObject ) :
 
 	__tabMimeType = "application/x-gaffer.compoundEditor.tab"
 
+	# Qt has a somewhat hard-coded handling of the drag/drop cursor which draws
+	# the 'forbidden' shape when outside of our windows. QDragManager also
+	# uses setOverrideCursor so setCursor doesn't have any effect. This is a
+	# nasty brute-force global event filter that attempts to wrangle back some
+	# control over the mouse cursor during a drag.
+	class _CursorBehaviour( QtCore.QObject ) :
```

Comment:
This guy isn't really working very well on my machine - it's flickering like crazy. I guess our only other option is the "initiate a gaffer drag programmatically" thing?

Reply:
Yeah, on Mac is a 3/10 on the crazy scale, at CS on linux it was 1/10... Looks like we should just do that programmatic drag...